src/lib/homestore_backend/resync_pg_shard.fbs

```diff
@@ -0,0 +1,25 @@
+namespace homeobject;
+
+table Member {
+   uuid : [ubyte];
+   name : [ubyte];
+}
+
+table PGInfoEntry {
+    pg_id : uint32;                     // only low 16 bit is used for pg_id;
+    replica_set_uuid : [ubyte];         // uuid of replica set
+    members : [Member];                 // peers;
```

Comment:
we need to add blob sequence number here or home object has to update , probably its on commit in its incremental resync.

Reply:
Isn't blob id (same as blob_seq_num) already included in BlobData already, it will update receiver's PG sb for every blob put? 
Why do we need another blob seq number, here? It will be a moving target for PG's blob seq id on leader side, rigth?

Reply:
yeah we have to update it in increment resync.